amt/api/routes/projects.py

```diff
@@ -27,8 +28,13 @@ async def get_new(
 async def post_new(
     request: Request,
     project_new: ProjectNew,
+    instrument_service: Annotated[InstrumentsService, Depends(InstrumentsService)],
+    task_service: Annotated[TasksService, Depends(TasksService)],
     projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
 ) -> HTMLResponse:
+    instruments = instrument_service.fetch_instruments(project_new.instruments)  # type: ignore
     project = projects_service.create(project_new)
+    for instrument in instruments:
```

Comment:
I would move the creation of tasks within the project_service.create method. That keeps the business logic together in 1 call.